packages/react/src/alert-dialog-title/AlertDialogTitle.tsx

```diff
@@ -23,12 +23,9 @@ export const AlertDialogTitle = forwardRef<
   const { restProps, sprinkleProps } = extractSprinkles(props);
   return (
     <Flex flexDirection="row" gap="md" p="lg" pb="md" {...sprinkleProps}>
-      {addonBefore && <Icon>{fallbackSpan(addonBefore)}</Icon>}
-      {!addonBefore && (
-        <Box asChild color="fg.error" h="16" w="auto">
-          <IconTriangleExclamation />
-        </Box>
-      )}
+      <Icon asChild color="fg.error">
```

Comment:
About the custom icon, would we always have the same colored? As CMP's icos are not in `red` color.

Reply:
it's fine to set it as a default - clients can re-color/re-size them as they require

Reply:
Got it, we updated Icon component with `asChild`, before it wasn't.